spec/services/bulk_upload/sales/year2023/row_parser_spec.rb

```diff
@@ -1008,6 +1008,60 @@ def purch_id_field
         end
       end
     end
+
+    describe "#field_105" do
+      let(:attributes) { valid_attributes.merge({ field_7: "1", field_105: "3" }) }
+
+      it "does not allow 3 (don't know) as an option for shared ownership" do
```

Comment:
I think this test case is current a bit misleading - I believe we currently do allow "don't know" for shared ownership under some conditions.

Reply:
Yep, I think the same thing is tested below, so removed this and tested the case where we do allow it